doc/cephadm/install.rst

```diff
@@ -30,7 +30,14 @@ The ``cephadm`` command can
 #. launch a containerized shell with a working Ceph CLI, and 
 #. aid in debugging containerized Ceph daemons.
 
-There are a few ways to install cephadm:
+There are two ways to install ``cephadm``:
+
+# a curl-based installation method
+# distribution-specific installation methods
```

Comment:
maybe [link](https://docutils.sourceforge.io/docs/user/rst/quickref.html#internal-hyperlink-targets) them?  

```suggestion
# a ´curl-based installation`_ method
# `distribution-specific installation`_ methods
```

Reply:
Not a bad suggestion. Done.